controllers/secret_controller.go

```diff
@@ -42,13 +44,15 @@ const (
 )
 
 // NewSecretReconciler returns a pointer to a SecretReconciler
-func NewSecretReconciler(mgr manager.Manager, watchNamespace string) *SecretReconciler {
+func NewSecretReconciler(mgr manager.Manager, clusterConfig cluster.Config, watchNamespace string) *SecretReconciler {
```

Comment:
This function requires only the platform itself, so it makes more sense to pass the platform in directly

Reply:
Addressed. Went this way for consistency with the signature of the other reconcilers.